indico/modules/rb/views/user/rooms.py

```diff
@@ -94,6 +94,25 @@ class WPRoomBookingSearchRoomsResults(WPRoomBookingBase):
     def __init__(self, rh, menu_item, **kwargs):
         self._menu_item = menu_item
         WPRoomBookingBase.__init__(self, rh, **kwargs)
+        self.addExtraJS('''
```

Comment:
Ok, I will do that right away. Since I don't know what is going to be kept in the new code I thought it is a good idea to utilize one of the functions, in this case `addExtraJS`.